website/docs/Selectors.md

```diff
@@ -505,17 +505,28 @@ browser.react$$('MyComponent') // returns the WebdriverIO Elements for the array
 If your app requires a specific way to fetch elements you can define yourself a custom selector strategy that you can use with `custom$` and `custom$$`. For that register your strategy once in the beginning of the test:
 
 ```js
-browser.addLocatorStrategy('myCustomStrategy', (selector) => {
-    return document.querySelectorAll(selector)
+browser.addLocatorStrategy('myCustomStrategy', (selector, root) => {
```

Comment:
Can't find anywhere in the code that this should work and when running this I also get a confirmation about this saying that root is undefined. Why is this change made?

Reply:
Also, if this is indeed true. We should also update the documentation at all other places and also our types to reflect this as well

Reply:
> Can't find anywhere in the code that this should work and when running this I also get a confirmation about this saying that root is undefined. Why is this change made?

It only works when calling the custom locator strategy on an element object. The reference implementation is here: https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/element/custom$.ts#L37-L51